docroot/themes/contrib/civictheme/src/CivicthemeUpdateHelper.php

```diff
@@ -0,0 +1,236 @@
+   *   The entity type manager.
+   * @param \Psr\Log\LoggerInterface $logger
+   *   Logger.
+   * @param int $batch_size
+   *   The number of entities to process in each batch.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerInterface $logger, $batch_size) {
+    $this->entityTypeManager = $entity_type_manager;
+    $this->logger = $logger;
+    $this->batchSize = $batch_size;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity_type.manager'),
+      $container->get('logger.factory')->get('action'),
+      self::BATCH_SIZE
+    );
+  }
+
+  /**
+   * Updates configuration entities as part of a Drupal update.
+   *
+   * @param array $sandbox
+   *   Stores information for batch updates.
+   * @param string $entity_type
+   *   Entity type to load.
+   * @param array $entity_bundles
+   *   Entity build to filter entities.
+   * @param callable $start_callback
+   *   Start callback function to call whne batch initialise.
+   * @param callable $process_callback
+   *   Process callback to process entity.
+   * @param callable $finished_callback
+   *   Finish callback called when batch finished.
+   */
+  public function update(array &$sandbox, $entity_type, array $entity_bundles, callable $start_callback, callable $process_callback, callable $finished_callback) {
+    $storage = $this->entityTypeManager->getStorage($entity_type);
+
+    // If the sandbox is empty, initialize it.
+    if (!isset($sandbox['entities'])) {
+      $sandbox['batch'] = 0;
+      $sandbox['current_entity'] = 0;
+      // Query to fetch all the manual_list paragraph ids.
+      $query = $storage->getQuery()->accessCheck(FALSE)
+        ->condition('type', $entity_bundles, 'in');
+      $sandbox['entities'] = $query->execute();
+      $sandbox['max'] = count($sandbox['entities']);
+
+      $sandbox['results']['processed'] = [];
+      $sandbox['results']['updated'] = [];
+      $sandbox['results']['skipped'] = [];
+
+      // Start callback.
+      call_user_func($start_callback, $this);
+    }
+
+    $sandbox['batch']++;
+
+    /** @var \Drupal\Core\Entity\EntityInterface $entity */
+    $entities = $storage->loadMultiple(array_splice($sandbox['entities'], 0, $this->batchSize));
+
+    foreach ($entities as $entity) {
+      $sandbox['results']['processed'][] = $entity->id();
+      $sandbox['current_enity'] = $entity;
+      // Process entity.
+      call_user_func($process_callback, $this, $entity);
+    }
+
+    $sandbox['#finished'] = empty($sandbox['entities']) ? 1 : ($sandbox['max'] - count($sandbox['entities'])) / $sandbox['max'];
+
+    if ($sandbox['#finished'] >= 1) {
+      $this->logger->notice("Update results ran in %batches batch(es):\n   Processed: %processed %processed_ids\n   Updated: %updated %updated_ids\n   Skipped: %skipped %skipped_ids\n", [
```

Comment:
@AlexSkrypnyk I have used the logger to log the results but also need this information on screen. Drush is showing these notices but not on webpage.

Reply:
@govindmaloo yeah, this is a known issue

can you please try using something like this
https://github.com/drevops/drevops/blob/9.x-1.14/docroot/modules/custom/your_site_core/src/PostConfigImportUpdateHelper.php#L34